src/packagedcode/npm.py

```diff
@@ -407,18 +459,23 @@ def licenses_mapper(license, licenses, package):  # NOQA
     - (Deprecated) an array or a list of arrays of type, url.
     -  "license": "UNLICENSED" means this is proprietary
     """
-    licensing1 = get_licensing(license)
-    licensing2 = get_licensing(licenses)
-    declared_license = '\n'.join([v for v in (licensing1, licensing2) if v])
+    # Value by parsing 'license' element
+    declared_license_via_license = get_declared_licenses(license)
+    # Value by parsing 'license' element
```

Comment:
remove this comment and the one above: this one is wrong we are parsing "licenses" (plural) and not "license"